common/static/common/js/spec/components/paging_collection.js

```diff
@@ -32,6 +32,44 @@ define(["backbone.paginator", "backbone"], function(BackbonePaginator, Backbone)
             this.currentPage = currentPage;
             this.start = start;
             return response.items;
+        },
+
+        setPage: function (page) {
```

Comment:
Sorry to have dropped this comment. We're working on getting rid of this weird mock collection and testing with the real one.